README.md

```diff
@@ -73,3 +73,4 @@ Supported Kubernetes versions:
   - 1.7.10
   - 1.8.0
   - 1.8.4
+  - 1.8.6
```

Comment:
I think this was added in kops version `1.8.1`, right? Same with the AMI changes.

Can you also bump the `Kops version:` in this README.md file and also in https://github.com/FutureSharks/tf-kops-cluster/blob/master/module/locals.tf#L3 otherwise there will be a mismatch 🙂

Reply:
Ah didn't realise there was a new Kops version. I've been using 1.8.0 with these settings. According to the version matrix all of 1.8.x will support all of k8s 1.8.x. so ideally we wouldn't lock the `supported_kops_version` to one version.